src/App.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
+import './App.css';
+import {ChatLinks} from './ChatLinks.js'
+import {ChatBody} from './ChatBody'
 
 function App() {
   return (
-    <div>Hello Pranit!!!</div>
+    <div className='app__main'>
+      <div className='app__body'>
+        <ChatLinks/>
+        <ChatBody/>
+      </div>
```

Comment:
Names of component not suitable
Use Sidebar instead of ChatLinks
Chat instead of ChatBody